src/containers/app/navigation/profile/SettingsDropdown/index.js

```diff
@@ -59,10 +61,10 @@ const SettingsDropdown = ({ logoutUser }) => {
   const menu = (
     <StyledMenu>
       <StyledMenuItem>
-        Account Profile
+        <StyledMenuButton type="text">Account Profile</StyledMenuButton>
```

Comment:
These would server better as links instead of buttons. Intercept link with event.preventDefault() and do what needs to be done.